src/simulator/AbstractGUI.java

```diff
@@ -8,6 +8,8 @@
 
 import java.awt.event.*;
 
+//TODO: there is a scrollpane (bool) and scrollPane (JScrollPane)
```

Comment:
The TODO doesn't say what needs to be done, it's a statement.  If there is work in the future, please describe what it is...if not, go ahead and remove it.